### ISSUE TITLE
Seems like the loss function plays a major role?

### ISSUE THREAD

**Issue by fayewong666999** [KaiChen1998/GeoDiffusion#24]
We followed your work and conducted further ablation experiments, that is, removing "area re-weighting" in loss, and found that the FID would be greatly improved (coco 256x256 FID:10.47), but the mAP also decreased a lot. What is the reason? Does this mean that "loss re-weighting" in your work is a major contributor to downstream performance?

**Comment by KaiChen1998** [comment]
Thank you for your comments!

It is a very interesting observation, which has also been discussed in our paper (Table 9 & Appendix B) that foreground re-weighting is trading image quality (i.e., FID) with annotation consistency (i.e., mAP).

However, both metrics are important for generated images to be useful in training object detectors, and higher mAP does not always result in better downstream **trainability** (which is the final purpose of introducing generated images) performance.

Therefore, I agree that "loss re-weighting" is an essential component of GeoDiffusion, but there indeed exists a balanced point among image quality and annotation consistency, which we can control via adjusting the hyper-parameters of "loss re-weighting" (as seen in Table 9).
